### PR TITLE
Docker refactor

### PR DESCRIPTION
**Pull request by khast3x**
Hey there,

I hope this works for you. Copy pasting the details from the first PR below:

Best,

---
Updated the Dockerfile to be much simpler, and to use a stable Metasploit+Postgresql Docker image as base instead of the previous awkward install.   

Since its now pre-configured, the Dockerfile is compatible for auto-building on Dockerhub at every push (it's free). I attached an example in the documentation using the build I did creating this fork. If you decide to use that feature and set it up, it'll be up to date every time you push :sunglasses:     

You'll just need to create an account, link github, and create a build for AutoSploit. Since the Dockerfile is in the `Docker/` folder, be sure to set it in the autobuild configuration [like so](https://i.postimg.cc/L5Zc3kbR/Screenshot-2019-04-12-09-24-14.png).  

Tried it on an EC2. The Datacenter speed+AutoSploit combo was pretty ridiculous :rocket: 

Finally, if you do decide to have an auto-build setup, I'll be sure to add AutoSploit to my [side project](https://github.com/khast3x/Redcloud). Trying out the AutoSploit container to Multi-Tor container, results are very nice.

Let me know if you have any questions/feedbacks before merging.

Regards,

### CODE REVIEW

**Comment by Ekultek** [comment]
Looks good to me @NullArray?

**Comment by NullArray** [comment]
Hey guys, thank you for the pull request @khast3x it's always appreciated bro.  I'll go over the commit real quick and add my review. After which i will merge it.

File: Docker/README.md [hunk 1]
```diff
@@ -1,75 +1,19 @@
 # Docker deployment instructions
```
Comment (NullArray) [inline Docker/README.md]: We should probably reflect the changes in the main README.md as well. Or at least leave a reference to the readme that currently lives in the docker directory.

Wiki entry might be appropriate as well.

Comment (NullArray) [reply Docker/README.md]: Alright i did the main README.md for now. Just wanted to post that here.

**Comment by khast3x** [comment]
You're welcome, was great seeing it roll smoothly with the clean MSF+DB base.  
Let me know if you make an auto-build, would be really neat!

Best,

**Comment by NullArray** [comment]
> You're welcome, was great seeing it roll smoothly with the clean MSF+DB base.
> Let me know if you make an auto-build, would be really neat!
> 
> Best,

Yeah man, for sure. If we make an auto-build you'll be the first to know :+1:

**Pull request merged**